distributions/standalone/src/main/resources/MIGRATION.txt

```diff
@@ -18,6 +18,7 @@ MULE-10979: The default response timeout and default transaction timeout can't b
 MULE-11118: The HTTP listener now replies with status code 503 when the thread pool is exhausted (and poolExhaustedAction="ABORT") instead of closing the socket.
 MULE-11825: In a DB template query, to set a DB param with the null value, you can use the "NULL" literal value. For example: <db:in-param name="name" defaultValue="NULL"/>
 MULE-12385: Reconnection Strategies can only be defined in connector components or globally (using <configuration> element). In mule 3.x, defining reconnection strategies was being supported by the XSD, but ignored by Runtime. Now, the XSD was changed to not allow to use this invalid configuration.
+MULE-12612: As FTP reconnection is at operation level, FTP connector does not support asynchronous reconnection strategies.
```

Comment:
Add more context, like explaining if this was working or not before and how to update the app if required